reggie/ingestion/preprocessor/washington_preprocessor.py

```diff
@@ -1,20 +1,25 @@
+import datetime
+import json
+
+from datetime import datetime
+from dateutil import parser
+from io import StringIO
+
+import numpy as np
+import pandas as pd
+
+from detect_delimiter import detect
```

Comment:
looks like this new import is causing test failures - you just need to add it to requirments.txt, I think?

Reply:
Should be added to the `setup.py` as well.